### DISCUSSION TITLE
How to pass an absolute path instead of relative path in file location.

### DISCUSSION THREAD

**Discussion by Sivasai2207** [exadel-inc/compreface-javascript-sdk#12] [Q&A]
![Screenshot 2021-05-27 at 7 58 04 PM](https://user-images.githubusercontent.com/49616092/119844569-2d9f2900-bf26-11eb-8667-e651a04f353f.png)

In the above screenshot line 3, I have tried to train a model by passing a relative path than in that case it works. But to automate the process using the Node Js File system module. It generates the absolute path so I need to pass the absolute path instead of the relative path.

I have tried implementing an absolute path instead of a relative path. I ran into an error. shows {
Oops! There is a problem in uploading image Error: Request failed with status code 400
}

![Screenshot 2021-05-27 at 8 00 09 PM](https://user-images.githubusercontent.com/49616092/119847066-4ad4f700-bf28-11eb-8b97-5d3ce7d4b23a.png)

**Comment by Dastnbek** [comment]
Hi @sivasai2207!
Yep. Unfortunately our sdk does not support absolute path in current version.
In your case, fast solution would be converting absoulute path to relative one.

```
var path = require('path');
console.log(path.relative('/foo/bar/baz', '/foo'));
```

Thanks for using our sdk.
